client/config/defaults.js

```diff
@@ -22,6 +26,7 @@ export const INDEX_OPTIONS = {
 // default load qty for product grid
 export const ITEMS_INCREMENT = 24;
 
+Session.setDefault("DEFAULT_APP_CONTAINER", DEFAULT_APP_CONTAINER);
```

Comment:
Don't use session vars in `default.js` we will most likely move away from this solution in the future. And you can override components using `replaceComponent`